docs/eventing-api.md

```diff
@@ -1811,6 +1968,29 @@ filter will be sent to the Subscriber. If not specified, will default to allowin
 </tr>
 <tr>
 <td>
+<code>filters</code><br/>
+<em>
+<a href="#eventing.knative.dev/v1.SubscriptionsAPIFilter">
+[]SubscriptionsAPIFilter
+</a>
+</em>
+</td>
+<td>
+<em>(Optional)</em>
+<p>Filters is an experimental field that conforms to CNCF CloudEvents Subscriptions
+API. It&rsquo;s An array of filter expressions that evaluates to true or false.
+If any filter expression in the array evaluates to false, the event MUST
+NOT be sent to the Subscriber. If all the filter expressions in the array
+evaluates to true, the event MUST be attempted to be delivered. Absence of
+a filter or empty array implies a value of true. In the event of users
+specifying both Filter and Filters, then the later will override the first.
+This will allow our users to try out the effect of the new filters field
+without compromising existing Filter objects and try it out on existing
+Trigger objects.</p>
```

Comment:
"object" appears twice in the same sentence, with 2 different meanings. Could we say "attribute-based Filter" instead of "Filter objects", or something along those lines?